src/ophys_etl/transforms/mesoscope_2p/tiff.py

```diff
@@ -262,11 +265,21 @@ def volume_scans(self):
 
     @property
     def plane_stride(self):
-        return len(self.plane_scans)
+        stride = 0
+        # only increment stride for unique zs, because same z end up in the
+        # the same page of the tiff - or at least did when I last tested
+        # that scenario in Scanimage
+        for z in np.unique(self.plane_scans):
+            if any([roi for roi in self.rois if roi.scanned_at_z(z)]):
+                stride += 1
+
+        return stride
 
     @property
     def volume_stride(self):
         stride = 0
+        # this logic is probably incorrect if there are 2+ volume scans
+        # with identical zs, but needs testing to confirm
```

Comment:
Was there a conclusion to the test suggested in this comment?

Reply:
See response to Dan here

https://github.com/AllenInstitute/ophys_etl_pipelines/pull/146#discussion_r573942840